samples/websocket-client/extension/index.ts

```diff
@@ -0,0 +1,23 @@
+import { NodeCG } from "nodecg/types/server";
+import { requireService } from "nodecg-io-core/extension/serviceClientWrapper";
+import { WSClientServiceClient } from "nodecg-io-websocket-client/extension";
+
+module.exports = function (nodecg: NodeCG) {
+    nodecg.log.info("Sample bundle for <the-service-name> started");
+
+    const service = requireService<WSClientServiceClient>(nodecg, "websocket-client");
+    service?.onAvailable((client) => {
+        nodecg.log.info("Client has been updated.");
+
+        const sock = client.getNativeClient();
+        sock.addListener("message", (code: number, data: string) => {
+            nodecg.log.info(`recieved message "${data}" with code: ${code}`);
+        });
+        setInterval(() => {
+            nodecg.log.info("Sending ping ...");
+            sock.send("ping");
+        }, 1000);
```

Comment:
I would prefer this to be more like 10 seconds so it doesn't spam the log as badly.
```suggestion
        }, 10000);
```